content/applications/sales/crm/track_leads/lead_scoring.rst

```diff
@@ -98,14 +115,14 @@ To reactivate automatic probability, click on the gear icon next to the probabil
 Assign leads based on probability
 =================================
 
-Odoo *CRM* can assign leads/opportunities to sales teams and salespeople based on specified rules.
+Odoo **CRM** can assign leads/opportunities to sales teams and salespeople based on specified rules.
 Create assignment rules based on the leads' probability of success to prioritize those that are
 more likely to result in deals.
 
 Configure rule-based assignment
 -------------------------------
 
-To activate *rule-based assignment*, navigate to :menuselection:`CRM --> Configuration -->
+To activate *rule-based assignment*, navigate to :menuselection:`CRM app --> Configuration -->
 Settings`, and activate :guilabel:`Rule-Based Assignment`.
 
 The rule-based assignment feature can be set to run :guilabel:`Manually`, meaning an Odoo user must
```

Comment:
```suggestion
The Rule-Based Assignment feature can be set to run :guilabel:`Manually`, meaning an Odoo user **must**
```
OPTIONAL: opting for capitalization here because it's the proper name of a feature

Reply:
If you decide to accept this suggestion, ensure that it is presented in the same way throughout the doc